File: static-analyzer/test-snippets/LinearSpaceAllocation.java
public class LinearSpaceAllocation {
    public static int[] duplicate(int[] arr) {
        int[] copy = new int[arr.length];
        for (int i = 0; i < arr.length; i++) {
            copy[i] = arr[i];
        }
        return copy;
    }

    public static void main(String[] args) {
        int[] data = {1, 2, 3};
        System.out.println(duplicate(data).length);
    }
}

File: static-analyzer/test-snippets-csharp/LinearSpaceAllocation.cs
public class LinearSpaceAllocation
{
    public static int[] Duplicate(int[] arr)
    {
        int[] copy = new int[arr.Length];
        for (int i = 0; i < arr.Length; i++)
        {
            copy[i] = arr[i];
        }
        return copy;
    }

    public static void Main()
    {
        int[] data = { 1, 2, 3 };
        System.Console.WriteLine(Duplicate(data).Length);
    }
}
